test/support/single.spec.js

```diff
@@ -1,10 +1,7 @@
 require('./../utils/starter.spec.js').runTests({
     description: 'In reference,',
     testCases: [
-        "/reference/tests/01.reference.js",
```

Comment:
Is this intentional or did you disable other tests during development and forget to reenable them?